vllm/logging/new_line_formatter.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+
+class NewLineFormatter(logging.Formatter):
```

Comment:
This is a bit awkward, but I needed to put this somewhere other than `vllm/logger` because otherwise it can't be auto-loaded during the loading of `vllm/logger` because while the module is loading, it can't be imported from. So `logging.config.dictConfig` wouldn't be able to load this formatter unless it lives not in `vllm/logger`.

Reply:
maybe just call the file formatter.py, so we can have more formatter later?

Reply:
Done :+1: